src/ui/components/Library/Team/View/Tests/TestListItem.tsx

```diff
@@ -1,20 +1,44 @@
-import { useContext } from "react";
+import { useContext, useMemo } from "react";
 
 import { Test } from "shared/test-suites/TestRun";
 import HighlightedText from "ui/components/Library/Team/View/TestRuns/HighlightedText";
 import Icon from "ui/components/shared/Icon";
 
 import { TestContext } from "./TestContextRoot";
-import styles from "../../../Library.module.css";
+import styles from "../NewTestRuns/TestRuns.module.css";
```

Comment:
I'm re-using what I defined for test runs but I saw frontend team deciding on new guideline but I was not sure if this the right way. So just pointing this out.

Reply:
I think we should move the relevant styles into a specific `.module.css` file for this component.

Reply:
That style is used in multiple component. Should we replicate same style for each component or create shared.module.css?

Reply:
@callingmedic911 Relevant comments @ 
* https://linear.app/replay/issue/FE-2120/add-top-level-testsuites-css#comment-1f7b6338
* https://linear.app/replay/issue/FE-2120/add-top-level-testsuites-css#comment-e334d01e